src/BlazorWebView/src/SharedSource/WebView2WebViewManager.cs

```diff
@@ -59,7 +59,7 @@ public class WebView2WebViewManager : WebViewManager
 		/// </summary>
 		protected static readonly string AppOrigin = $"https://{AppHostAddress}/";
```

Comment:
This could be made private now.

Reply:
I believe this entire type will soon be internal so that isn't necessary anymore 😄